### PR TITLE
Adds thank you message, radio button pushability, and better tabbing …

### LINKED ISSUE #332: "Thank you for your feedback" message is not displayed after the user unsubscribes
[Affected versions]:
- Firefox 62.0
- Firefox 63.0a1

[Affected Platforms]:
- Windows 10 x64
- macOS 10.13
- Ubuntu 16.04 x64

[Prerequisites]:
You have to be subscribed to Firefox Monitor and have the Unsubscribe page loaded. 

[Steps to reproduce]:
1. Click on the "Unsubscribe" button. 

[Expected result]:
- The user is redirected to "Unsubscribe Confirmation Survey" and then a "Thank you for your feedback" message is displayed.
- https://mozilla.invisionapp.com/share/CBNKNFNJ8E6#/screens/317396218

[Actual result]:
- The user is redirected to the Firefox Monitor home page.

[Regression]:
- I don't think this is a regression.

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
…to unsubscribe form.

Fixes #332

### CODE REVIEW

File: public/js/monitor.js [hunk 1]
```diff
@@ -193,7 +193,7 @@ async function hashEmailAndSend(emailFormSubmitEvent) {
     emailForm.querySelector("input[name=emailHash]").value = await sha1(emailInput.value);
     emailInput.value = "";
   }
-  emailForm.submit();
+  // emailForm.submit();
```
Comment (groovecoder) [inline public/js/monitor.js]: Do we need this line at all?

File: public/js/monitor.js [hunk 2]
```diff
@@ -226,29 +226,53 @@ const postData = (url, data = {}) => {
...
+  additionalBreaches.classList.toggle("show-breaches");
+  //setting height this way enables transition easing... setting the new height to "auto" 
+  if (additionalBreaches.classList.contains("show-breaches")) {
+    additionalBreaches.style.height = additionalBreaches.scrollHeight + "px";
+  }
+}
+
+const handleRadioButtons = function(form) {
+  const inputFields = form.querySelectorAll(".radio-button-group, .button");
+  inputFields[0].querySelector("input").checked = true;
```
Comment (groovecoder) [inline public/js/monitor.js]: Do we need a choice to be pre-checked? It may lead users to over-select this choice.

Comment (lesleyjanenorton) [reply public/js/monitor.js]: This was in accordance with a Photon radio-button stipulation that one should always be checked, but I wondered about over-selection/lazy-selection also.

File: public/js/monitor.js (hunk 1 above)
Comment (lesleyjanenorton) [reply public/js/monitor.js]: Yes. Particularly. Uncommenting!

File: public/js/monitor.js [hunk 3]
```diff
@@ -226,29 +226,53 @@ const postData = (url, data = {}) => {
...
+  for (let x = 0; x < inputFields.length ; x++) { 
+    const input = inputFields[x];
+    input.addEventListener("focus", (e) => {
+      input.addEventListener("keydown", (e) => {
```
Comment (pdehaan) [inline public/js/monitor.js]: Do we need to remove the `keydown` listener anywhere?
Like, if we focus the input 3 times, will it add the `keydown` event listener 3 times?

File: public/js/monitor.js (hunk 2 above)
Comment (groovecoder) [reply public/js/monitor.js]: Let's remove it ... that may apply more/better in Firefox UI where we have a good idea of sensible defaults. Here we'd like to keep it open-ended.

**Comment by lesleyjanenorton** [comment]
Added keyCode commenting, removed default selected radio button, and added error message on submissions with no option selected.

**Pull request merged**